commands/command_clean.go

```diff
@@ -77,8 +77,9 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 func init() {
 	RegisterSubcommand(func() *cobra.Command {
 		return &cobra.Command{
-			Use: "clean",
-			Run: cleanCommand,
+			Use:    "clean",
+			Run:    cleanCommand,
+			PreRun: resolveLocalStorage,
```

Comment:
I think I am going to leave this as-is, and only give the PreRun option to commands that need it.